src/pages/Details.vue

```diff
@@ -354,9 +340,23 @@ export default {
             return getResBaseURL() + this.monitor.screenshot + "?time=" + this.cacheTime;
         }
     },
+
+    watch: {
+        page(to) {
```

Comment:
Good point. This code normally works because AFAIK in the vue lifecycle computed runs before mounted, so `this.monitor` should be true. I added a watch for `monitor` in case the monitorList has not been loaded when we are mounted for some reason.